src/app/modules/main/wallet_section/activity/entry.nim

```diff
@@ -176,6 +162,7 @@ QtObject:
       return ""
     return self.transaction[].getNFTName()
 
+  # TODO: Any way to provide this in activity history service without fetching the whole transaction?
```

Comment:
@dlipicar I think you can help best answer this question

Reply:
ha! Let me merge my [collectibles improvements PR](https://github.com/status-im/status-go/pull/3734) first. Once that's done, my suggestion would be to have status-go provide NFT-related data as part of the activity metadata

Reply:
Oh wait there's actually no reason why we cannot do that now, since my changes are mostly related to ownership deta and not NFT metadata

Reply:
Will follow up with this change with work for https://github.com/status-im/status-desktop/issues/11597